javascript_nodejs/5.multiturn-prompts-bot/dialogs/mainDialog/index.js

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActionTypes, MessageFactory } = require('botbuilder');
+
+const { TextPrompt, NumberPrompt, DialogSet, WaterfallDialog } = require('botbuilder-dialogs');
+
+const DIALOG_STATE_PROPERTY = 'dialogState';
+const USER_NAME_PROP = 'user_name';
+const AGE_PROP = 'user_age';
+const WHO_ARE_YOU = 'who_are_you';
+const HELLO_USER = 'hello_user';
+
+const NAME_PROMPT = 'name_prompt';
```

Comment:
>name_prompt [](start = 21, length = 11)

Why do these two have string values that end in _prompt, but the PROP ones don't end in "_property"? (real question)

Reply:
I was thinking that the property names mapped to field names in the model, so we probably wouldn't want to include "_property" in them.  Am I wrong?

I didn't think TOO hard about what those values where - is there a pattern that has emerged?